src/index.html

```diff
@@ -11,7 +11,171 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="field_set">
+        <legend>Personal information</legend>
+        <label class="form_field">
+          Surname:
+          <input type="text" 
+          name="surname"
+          autocomplete="off" 
+          required
+          class="field"
+          >
```

Comment:
```suggestion
          <input 
            type="text" 
            name="surname"
            autocomplete="off" 
            required
            class="field"
          >
```
Каждый атрибут с новой строки,2 отступа,закрывающяя и открывающая дужка тега на одном уровне